projects/pgai/tests/vectorizer/test_vectorizer_cli.py

```diff
@@ -235,23 +245,41 @@ def test_params(request: pytest.FixtureRequest) -> tuple[int, int, int, str, str
 
 class TestWithOpenAiVectorizer:
     @pytest.mark.parametrize(
-        "test_params",
+        "test_params,openai_custom_base_url",
         [
             (
-                1,
-                1,
-                1,
-                "chunking_character_text_splitter('content')",
-                "formatting_python_template('$chunk')",
+                (
+                    1,
+                    1,
+                    1,
+                    "chunking_character_text_splitter('content')",
+                    "formatting_python_template('$chunk')",
+                ),
+                None,  # No base_url is set. Use default (https://api.openai.com/v1)
```

Comment:
This parameter needs to flow into the casette name

Reply:
Good point. Added 👍